src/briefcase/platforms/iOS/xcode.py

```diff
@@ -387,6 +389,8 @@ def _install_app_requirements(
             pip_args=[
                 f"--platform=ios_{ios_min_tag}_{self.tools.host_arch}_iphonesimulator",
             ],
+            install_hint="It may also indicate that an `iphoneos` wheel could be found, "
+            "but an `iphonesimulator` wheel could not be found.",
```

Comment:
As above, ordering of arguments needs tweaking. There's also a need to add extra whitespace at the start:
```suggestion
            install_hint="""

It may also indicate that an `iphoneos` wheel could be found, but an
`iphonesimulator` wheel could not be found.
"""
```